src/components/geo-layer-control.ts

```diff
@@ -0,0 +1,65 @@
+import { html, LitElement, unsafeCSS } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import Control from 'ol/control/Control';
+
+import style from '../styles/svg-control.css?inline';
+import control from '../styles/controls.css?inline';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import SVGCreator from '../utils/svg-creator';
+
+import { useStore } from '../composable/store';
+import LayerSelectionControl from './layer-selection';
+
+@customElement('geo-layer-control-button')
+class GeoLayerControlButton extends LitElement {
+  static styles = [unsafeCSS(style), unsafeCSS(control)];
+
+  render() {
+    return html`<div class="ol-unselectable ol-control center-control">
+                  <div>
+                    <div class="control-${useStore().getTheme()}">
+                      ${unsafeSVG(SVGCreator.stack)}
+                    </div>
+                  </div>
+                </div> `;
+  }
+}
+
+export default class GeoLayerControl extends Control {
+
+  isOpen: Boolean = false;
+  layerSelection: LayerSelectionControl;
+
+  constructor(target: HTMLElement) {
+    const element = document.createElement(
+      'geo-layer-control-button'
+    ) as GeoLayerControlButton;
+
+    super({
+      element: element,
+    });
+
+    this.layerSelection = new LayerSelectionControl();
+    this.layerSelection.disable();
```

Comment:
why do you disable it?

Reply:
Because it's the layer selection menu and not the control button and by default, the selection menu is disabled